src/utils/sdkApiError.ts

```diff
@@ -46,7 +45,12 @@ export class sdkApiError extends Error {
     this.status = customError.status;
     this.statusText = customError.statusText;
     this.hoprdErrorPayload = customError.hoprdErrorPayload;
-    this.description = customError?.hoprdErrorPayload?.error ? customError.hoprdErrorPayload.error : `HTTP Status code ${customError.status}`;
+    this.description = `HTTP Status code ${customError.status}`;
+    if(customError.hoprdErrorPayload?.status) this.description = customError.hoprdErrorPayload.status;
```

Comment:
Looks to me as if linting is disabled??

Reply:
The rules might not be the best:
![image](https://github.com/user-attachments/assets/f75efd54-7cd2-446c-8151-9cc6a7fe858e)

Reply:
Will enable it with full set of linting next
https://github.com/hoprnet/hopr-sdk/issues/126